app/ts/SharedUI/YesNoCategoricalMarketReportingOptions.tsx

```diff
@@ -9,7 +9,7 @@ import { MarketData } from './Market.js'
 export type OutcomeStake = {
 	outcomeName: string
 	repStake: bigint
-	status: 'Winning' | 'Losing'
+	status: 'Winning' | 'Losing' | 'Tie'
```

Comment:
Isn't there always a winner at any given point in time except at the very start before reporting?  If the two values are equal, isn't the first one to reach that value the "winner" until the other passes it up?